HOWTOUSE.md

```diff
@@ -84,6 +86,42 @@ And deactivate it just by running this command on the terminal
 $ deactivate
 ```
 
+
+### (Ana)Conda
+
+If you prefer, you can create a virtual environment using conda.
+
+Create the environment as follows:
+
+```bash
+$ conda install --name [the name of the environment] python=3.5
+```
+
+Afterwards, setup the scripts for activating the environment variables, and unsetting them when we activate and deactivate it.
+
+Locate the directory of your environment such as ```bash /anaconda/envs/[name_environment]``` and create the following subdirectories:
+
+```bash
+cd /anaconda/envs/[name_environment]
+mkdir -p ./etc/conda/activate.d
+mkdir -p ./etc/conda/deactivate.d
+touch ./etc/conda/activate.d/env_vars.sh
+touch ./etc/conda/deactivate.d/env_vars.sh
+```
+
+Then, use ```activate.d/env_vars.sh``` to set the necessary environment variables, and ```deactivate.d/env_vars.sh``` to unset them.
```

Comment:
Mas bien, solo se deben de modificar esos archivos para: "to set the necessary environment variables." Creo que tal vez valdría la pena especificar que se modifican esos archivos solo para definir las variables. A mi me quedo claro pero es porque uso anaconda.

Cuando corres source activate corre esos scripts.